src/introduction/index.tex

```diff
@@ -0,0 +1,12 @@
+% textidote: ignore begin
+\chapter{Introduction}\label{ch:introduction}
+% textidote: ignore end
+
+Business intelligence is important for any business.
```

Comment:
Use `\acrfull` instead of plaintext

Reply:
@Danar435 See my comment on your own PR

Reply:
Using \acrfull will not allow for a capital B and a capital B is needed as it is the beginning of the sentence, which is why I did it like this:)

Reply:
@MadsHeilmann @Danar435 I think the point of the acronyms package is to insert real acronyms mid sentence, where all the words of the acronym would usually be spelled out with all lowercase letters. It's not a database for "saving" reusable phrases or words. That's why overusing `\acrfull` is not desirable. See the documentation for examples.